src/main/java/de/unijena/cheminf/mortar/gui/views/SettingsView.java

```diff
@@ -312,6 +314,18 @@ protected void updateItem(IDisplayEnum item, boolean empty) {
                 default ->
                         throw new UnsupportedOperationException("Unknown property type " + tmpProperty.getName());
             }
+
+            Region backgroundRegion = new Region();
+            backgroundRegion.setPrefHeight(40);
```

Comment:
this contradicts line 200 "tmpRow.setPrefHeight(50);", doesn't it? I think the row and its background should have the same preferred height (not sure which one is trumping the other right now).